spec/gitlab/client/repositories_spec.rb

```diff
@@ -132,7 +132,7 @@
 
   describe '.generate_changelog' do
     before do
-      stub_post('/projects/3/repository/changelog', 'changelog')
+      stub_post('/projects/3/repository/changelog', 'generate_changelog')
```

Comment:
why was this changed?

Reply:
For stub response filenames to better correspond to method names in which test they are used.

BTW, I was the one who also added original `generate_changelog` method.